packages/g/gendesk/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : gendesk
+version    : 1.0.10
+release    : 1
+source     :
+    - https://roboticoverlords.org/gendesk/gendesk-1.0.10.tar.xz : 315cc9c348aef85eedff39c774b03a8ac5341246ac3bb5036566ff7ae7d8dae8
+    - https://roboticoverlords.org/images/gendesk.png : 4d96eded48e536d02e35727c36dc20844c2e44654e81baf78e10aee4eb48e837
+homepage   : https://gendesk.roboticoverlords.org/
+license    : MIT
+summary    : Utility for generating desktop files by specifying a minimum of information
+description: |
+    Utility for generating desktop files by specifying a minimum of information
+builddeps  :
+    - golang
+build      : |
+    go build -v -mod=vendor -trimpath -buildmode=pie -ldflags="-s -w -extldflags '$LDFLAGS'"
+install    : |
+    install -Dm755 gendesk "$installdir/usr/bin/gendesk"
```

Comment:
You can remove the quotes: `install -Dm755 gendesk "$installdir/usr/bin/gendesk"` -> `install -Dm755 gendesk $installdir/usr/bin/gendesk`
Please do this for all install commands here